code/pizzabot.js

```diff
@@ -5,3 +5,76 @@ const pepperoni = "Pepperoni Pizza"
 const pizzaPrice = 80
 
 //Put your Javscript code here:
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
```

Comment:
The `console.log`'s is something we use when we are develop the things. So it's only for you to check so everything is correct. When doing a pr you can delete it!